helpdesk_mgmt/data/helpdesk_data.xml

```diff
@@ -28,39 +28,45 @@
       <field name="code">helpdesk.ticket.sequence</field>
       <field name="prefix">HT</field>
       <field name="padding">5</field>
+      <field name="company_id" eval="False"/>
```

Comment:
Why not `<field name="company_id"/>` in all for saving characters?

Reply:
I'm not completely sure if I understand what you mean:

In the "ir.sequence" record I have added 'eval="False"' because it is how I have seen that Odoo does it in their code. Example:
https://github.com/odoo/odoo/blob/a3dcde21d51767d7ca9ab85da10e25ef24294248/addons/account/data/account_data.xml#L54

The rest of records are done the way I understand it has to be done, as well as you are saying, don't you?

Reply:
Yes, all works the same way. My proposal is for 3 things:

* Homogeneity for all the assignments. You are using 2 techniques.
* Clearer syntax: IMO this seems clearer (but it's subjective).
* Save code characters. Digital kittens will thank you :yum:

Reply:
Do you agree then?

Reply:
Yes, Im changing from

`<field name="company_id" eval="False"/>` 

to 

`<field name="company_id"/>`

right now.

Reply:
And the other occurrences of `<field name="company_id"></field>` to `<field name="company_id"/>`, please.